src/jit/gschecks.cpp

```diff
@@ -413,7 +413,9 @@ void Compiler::gsParamsToShadows()
             lvaTable[shadowVar].lvBaseType  = varDsc->lvBaseType;
         }
 #endif
+
         lvaTable[shadowVar].lvRegStruct = varDsc->lvRegStruct;
+        lvaTable[shadowVar].lvExactSize = varDsc->lvExactSize;
```

Comment:
> lvaTable[shadowVar].lvExactSize = varDsc->lvExactSize; [](start = 7, length = 55)

This is already under above under #ifdef FEATURE_SIMD.

In fact initializing lvExactSize for shadow vars other than those known to be SIMD has caused a gc-hole.

Why do we need this change, btw?